main.js

```diff
@@ -79,7 +79,7 @@ var handleStartupEvent = function() {
 };
 
 if (handleStartupEvent()) {
-  return;
```

Comment:
Temporarily put this in, because the client was bombing on here (when not using the installer). @hoffmabc is looking into a fix.